client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -393,6 +393,7 @@ endif()
 find_package(Boost 1.54 REQUIRED COMPONENTS log system thread)
 find_package(Avro REQUIRED)
 find_package(Botan REQUIRED)
+find_package(Threads REQUIRED)
```

Comment:
Have you investigated, why the
_find_package(Boost 1.54 REQUIRED COMPONENTS log system **thread**)_
cannot find this library on Intel Edison?